openid-connect-server/src/test/java/org/mitre/oauth2/service/impl/TestDefaultIntrospectionResultAssembler.java

```diff
@@ -302,7 +306,75 @@ public void shouldAssembleExpectedResultForRefreshTokenWithoutUserAuthentication
 				.build();
 		assertThat(result, is(equalTo(expected)));
 	}
+	
+	
+	@Test(expected= ParseException.class)
+	public void testAssembleFrom_OAuth2AccessTokenEntityExpiration() throws ParseException{
+		
+		// given
+				OAuth2AccessTokenEntity accessToken = accessToken(null, scopes("foo", "bar"), null, "Bearer",
+						oauth2AuthenticationWithUser(oauth2Request("clientId"), "name"));
+
+				UserInfo userInfo = userInfo("sub");
+
+				Set<String> authScopes = scopes("foo", "bar", "baz");
+
+				// when
+				Map<String, Object> result = assembler.assembleFrom(accessToken, userInfo, authScopes);
+
+
+				// then
+				Map<String, Object> expected = new ImmutableMap.Builder<String, Object>()
+						.put("sub", "sub")
+						.put("exp",123L)
+						.put("scope", "bar foo")
+						.put("active", Boolean.TRUE)
+						.put("user_id", "name")
+						.put("client_id", "clientId")
+						.put("token_type", "Bearer")
+						.build();
+				
+				
+				doThrow(ParseException.class).when(accessToken).getExpiration();
+				accessToken.getExpiration();
```

Comment:
This is a pointless test. You're telling the mock to throw an exception when you make a call and then calling that function explicitly. 

What are you trying to accomplish here?